shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -185,15 +187,43 @@ - (void)testStartKeyboardAnimationWillInvokeSetupKeyboardAnimationVsyncClient {
   OCMVerify([viewControllerMock setupKeyboardAnimationVsyncClient]);
 }
 
-- (void)testkeyboardWillChangeFrameWillStartKeyboardAnimation {
+- (void)testKeyboardWillShowNotificationEmptyKeyboard {
   FlutterEngine* mockEngine = OCMPartialMock([[FlutterEngine alloc] init]);
   [mockEngine createShell:@"" libraryURI:@"" initialRoute:nil];
   FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:mockEngine
                                                                                 nibName:nil
                                                                                  bundle:nil];
+  // keyboard is empty
+  CGRect keyboardFrame = CGRectMake(0, 0, 0, 0);
+  BOOL isLocal = YES;
+  NSNotification* notification = [NSNotification
+      notificationWithName:@""
+                    object:nil
+                  userInfo:@{
+                    @"UIKeyboardFrameEndUserInfoKey" : [NSValue valueWithCGRect:keyboardFrame],
+                    @"UIKeyboardAnimationDurationUserInfoKey" : [NSNumber numberWithDouble:0.25],
+                    @"UIKeyboardIsLocalUserInfoKey" : [NSNumber numberWithBool:isLocal]
+                  }];
+  FlutterViewController* viewControllerMock = OCMPartialMock(viewController);
+  viewControllerMock.targetViewInsetBottom = 10;
 
-  CGFloat width = UIScreen.mainScreen.bounds.size.width;
-  CGRect keyboardFrame = CGRectMake(0, 100, width, 400);
+  [viewControllerMock keyboardWillShowNotification:notification];
+  XCTAssertTrue(viewControllerMock.targetViewInsetBottom == 0);
+  OCMVerify([viewControllerMock startKeyBoardAnimation:0.25]);
+}
+
+- (void)testKeyboardWillShowNotificationKeyboarBelowScreen {
```

Comment:
nits:`testKeyboardWillShowNotificationKeyboarBelowScreen` should be `testKeyboardWillShowNotificationKeyboardBelowScreen ` (keyboard)  ^_^